core/templates/dev/head/pages/collection_editor/collection-editor-navbar/collection-editor-navbar.directive.ts

```diff
@@ -114,7 +115,7 @@ oppia.directive('collectionEditorNavbar', [
             var modalInstance = $uibModal.open({
               templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
                 '/pages/collection_editor/' +
-                'collection_editor_save_modal_directive.html'),
+                'collection-editor-save-modal.directive.html'),
```

Comment:
Since this is not the main HTML file of collection editor, please make a separate subfolder of these directive HTML files. You can see an example [here](https://github.com/YashJipkate/oppia/tree/modularity-restructure/core/templates/dev/head/pages/skill-editor-page/skill-editor-templates).

Reply:
Alright, ill do that once the e2e tests pass, I forgot to add the folder in the path.